packages/mobile/src/verify/VerificationEducationScreen.tsx

```diff
@@ -88,9 +92,18 @@ function VerificationEducationScreen({ route, navigation }: Props) {
   const currentState = useSelector(currentStateSelector)
   const shouldUseKomenci = useSelector(shouldUseKomenciSelector)
   const verificationStatus = useSelector(verificationStatusSelector)
+  const biometryEnabled = useSelector(biometryEnabledSelector)
   const choseToRestoreAccount = useSelector(choseToRestoreAccountSelector)
   const registrationStep = useRegistrationStep(route.params?.choseToRestoreAccount ? 4 : 3)
 
+  const handleProceedToNextStep = () => {
+    if (biometryEnabled) {
+      navigate(Screens.EnableBiometry)
```

Comment:
So biometry goes after phone number verification? I think if the user does verify their phone number then they never get to see the biometry screen, right? I think this could be added in `OnboardingSuccessScreen`

I think it would be better to do it before though, since phone number verification can be quite long it would make sense to not add extra steps after that, but if this has already been discussed and decided it's fine.

Reply:
oh Gonza, this was such a good catch. both Nitya and I forgot about this `OnboardingSuccessScreen` 🙈

i spoke with Nitya and have moved this screen to after the pincode screen. this brought an annoying problem of how to keep track of all the steps (not just the total steps, but also which step the screen is!) - i updated the useRegistrationStep hook, it's very verbose and i'm not sure i like it but my intention is for screens not to need to know too much about the number. really wanted to avoid the phone verification/import wallet screens needing to know about biometrics.....let me know what you think?

i'd hoped to get rid of `totalRegistrationStepsSelector` actually, but `PincodeSet` and `VerificationInputScreen` are still classes and can't use the `useRegistrationStep` hook 😭